docs/rules/keyword-spacing.md

```diff
@@ -80,59 +115,59 @@ if (foo) {
 } else {
     //...
 }
+```
 
-try {
+Examples of **correct** code for this rule with the `{ "before": false }` option:
+
+```js
+/*eslint keyword-spacing: ["error", { "before": false }]*/
+
+if (foo) {
     //...
-} catch (e) {
+}else if (bar) {
+    //...
+}else {
     //...
 }
+```
 
-switch (a) {
-    case +1:
-        break;
-}
+### after
 
-function foo() {
-    return [0, 1, 2];
-}
+Examples of **incorrect** code for this rule with the default `{ "after": true }` option:
+
+```js
+/*eslint keyword-spacing: ["error", { "after": true }]*/
 
-for (let [a, b] of [foo, bar, baz]) {
+if(foo) {
+    //...
+} else if(bar) {
+    //...
+} else{
     //...
 }
-
-let obj = {
-    get [FOO]() {
-        //...
-    },
-    set [FOO](value) {
-        //...
-    }
-};
-
-import {foo} from "foo";
-import * as bar from "foo";
 ```
 
-This rule is designed carefully to not conflict with other spacing rules.
```

Comment:
To understand the examples of correct code, this pair of sentences is needed as a second paragraph under Rule Details. To avoid 3 strikes with “this rule …” here is a possible rewording:

This rule is designed carefully not to conflict with other spacing rules: it does not apply to spacing where other rules report problems.